packages/editor/src/components/media-upload/README.md

```diff
@@ -101,6 +101,17 @@ CSS class added to the media modal frame.
 - Type: `String`
 - Required: No
 
+
+### addToGallery
+
+If true the gallery media modal will open in directly in the media library where additional images can be added.
+If false the gallery media modal will open in the edit mode where existing images can be edited.
+Only applies if gallery = true.
```

Comment:
Leaving this comment very much in passing, without looking much at context: The interface of this component doesn't otherwise reference "gallery" terminology, but rather "multiple". There is no "gallery = true"

Reply:
Hi @aduth, gallery and multiple are not the same thing. Multiple allows the selection of multiple files using the normal media modal frames, while the gallery prop opens specific gallery frames optimized for the gallery usage.
The gallery prop is not something we are adding here it is something that already existed before:
https://github.com/WordPress/gutenberg/blob/9a93e6a49fa86805e9e0a8bed6f102787e4d2e36/packages/block-library/src/gallery/edit.js#L200-L213


But, the documentation of the interface missed the gallery prop so it was misleading, I added the documentation to this prop.